hooks/useLogin.ts

```diff
@@ -70,54 +71,44 @@ export function useLogin(config: LoginProps = {}) {
           getUserDetails(token),
         ])
 
-        if ('error' in details || details instanceof LocalError) {
-          $setStatus(STATUS.FAILED)
+        if (details instanceof LocalError) {
```

Comment:
The first check doesn't do anything, instanceof tells TSC that it's the proper type with the right fields.